docs/preview/02-Features/05-AzureAppConfiguration/auto-refresh-app-configuration.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Automatically refresh Azure App Configuration"
+layout: default
+---
+
+# Automatically refresh Azure App Configuration
+The `Arcus.BackgroundJobs.AppConfiguration` library provides a background job to automatically refresh [configuration values](https://docs.microsoft.com/en-us/dotnet/core/extensions/configuration) in your application that comes from the Azure App Configuration.
+
+## Installation
+To use these features, you have to install the following package:
+
+```shell
+PM > Install-Package Arcus.BackgroundJobs.AzureAppConfiguration
+```
+
+## Setup
+To automtically notify your `IConfiguration` instance in your application for any change in the remote Azure App Configuration, you need some setup.
+We use the events from Azure App Configuration which will be send towards an Azure Service Bus Topic. The background job will look for those events on the topic, and will in turn refresh the `IConfiguration`.
```

Comment:
Is this done via EventGrid ?

Reply:
Might be nice if we have a graphical overview of how this works, and how the different components work together ?

Reply:
Yeah, we could do that. But I don't know where the Azure icons are, that we can use for this. 😅 The official package doesn't include Azure App Configuration.

Reply:
We could probably also benefit from a ARM template to generate this setup?

Reply:
Added an overview pic, https://github.com/arcus-azure/arcus.backgroundjobs/pull/84/commits/d3a7b50560f05ef020cfebb555d04eb8aa618318. Made sure it has a white background so it's still visible in a night-themed version of the site.

Reply:
Great thanks!